source/write/run-command.txt

```diff
@@ -0,0 +1,74 @@
+.. _java-rs-run-command:
+
+======================
+Run a Database Command
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: database operation, code example
+
+Overview
+--------
+
+In this guide, you can learn how to run a database command with the
+{+driver-short+}. Database commands can perform a variety of administrative and
+diagnostic tasks, such as fetching server statistics, initializing a replica
+set, or running an aggregation pipeline. You can use database commands when the
+driver does not contain a helper method for the command you want to run.
+
+.. important:: Prefer Driver Methods Over Database Commands
```

Comment:
S: so that you dont stack two important admonitions, maybe this one can be a tip instead